business_requirement/models/project.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    br_ids = fields.One2many(
+        comodel_name='business.requirement',
+        inverse_name='project_id',
+        string='Business Requirement',
+        copy=False,
+    )
+    br_count = fields.Integer(
+        compute='_compute_br_count',
+        string="Business Requirement Number"
```

Comment:
s/Business Requirement Number/Number of Business Requirements or Business Requirements count/